src/facade/dragonfly_connection.cc

```diff
@@ -300,8 +306,8 @@ Connection::Connection(Protocol protocol, util::HttpListenerBase* http_listener,
   id_ = next_id.fetch_add(1, memory_order_relaxed);
 
   queue_backpressure_ = &tl_queue_backpressure_;
-  if (queue_backpressure_->limit == 0) {
-    queue_backpressure_->limit = absl::GetFlag(FLAGS_pipeline_queue_limit);
+  if (queue_backpressure_->pipeline_queue_limit == 0) {
+    queue_backpressure_->pipeline_queue_limit = absl::GetFlag(FLAGS_pipeline_queue_limit);
     queue_backpressure_->pipeline_cache_limit = absl::GetFlag(FLAGS_request_cache_limit);
   }
```

Comment:
Should we rename all our flags to pubsub instead of just pipeline? Nobody is using them, but still if we decide to switch back later...?

Reply:
yes, I think we should. `subscriber_thread_max_bytes` ?

Reply:
And in general we should revisit the name of our limits, to make them more clear and consistent.

Reply:
Yes, I even like subscriber more than pubsub because we might reuse backpressure